pkg/tenant/db/members.go

```diff
@@ -22,29 +22,35 @@ const (
 )
 
 type Member struct {
-	OrgID        ulid.ULID    `msgpack:"org_id"`
-	ID           ulid.ULID    `msgpack:"id"`
-	Email        string       `msgpack:"email"`
-	Name         string       `msgpack:"name"`
-	Role         string       `msgpack:"role"`
-	Status       MemberStatus `msgpack:"status"`
-	Created      time.Time    `msgpack:"created"`
-	Modified     time.Time    `msgpack:"modified"`
-	DateAdded    time.Time    `msgpack:"date_added"`
-	LastActivity time.Time    `msgpack:"last_activity"`
+	OrgID        ulid.ULID `msgpack:"org_id"`
+	ID           ulid.ULID `msgpack:"id"`
+	Email        string    `msgpack:"email"`
+	Name         string    `msgpack:"name"`
+	OrgName      string    `msgpack:"org_name"`
+	OrgDomain    string    `msgpack:"org_domain"`
```

Comment:
We probably will need a migration to populate this information because this will require that users who have already registered go through the onboarding process again since we won't have the segment information (unless we pre-populate that information during the migration).

Reply:
Actually on second thought we should pre-populate all of the onboarding information for existing users; defaulting to "Work" for the professional segment and "Unspecified" for the developer segment; and they can edit this in their profile later; but it will ensure that the user is marked as onboarded.